alpha/apps/kaltura/lib/KSecureEntryHelper.class.php

```diff
@@ -102,13 +102,22 @@ public function __construct(entry $entry, $ksStr, $referrer, $contexts = array()
 		$this->applyContext();
 	}
 	
-	public function hasRules($contextType = null)
+	public function hasRules($contextType = null, $actionTypes = null)
```

Comment:
Why are you passing action type as array while its content is not referred (it is type boolean)?